### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

**Comment by jonsterling** [comment]
Love the unicode! Can't tell you how much abuse I took at my last job for using `-XUnicodeSyntax`, haha :smile:

**Comment by garyb** [comment]
Hah, I think we're all pretty happy about it except maybe @beckyconning :wink:

I hooked up my editor to automatically replace the common stuff as I type now, it's pretty sweet.

**Comment by jonsterling** [comment]
@garyb Which editor do you use? I was thinking about hacking together something for my Vim config to help with this.

By the way, I'd just say, I don't think anybody should be forced to use it if they find it too difficult to configure their editor—but I think it looks nice and it helps with alignment in many cases. My perspective is we should at least be consistent within a module/file.

**Comment by garyb** [comment]
This is in Sublime. It's pretty hacky, I just set it up to recognize key sequences like `"-", ">", " "` when editing `.purs` files, so there's a chance of some false positives. I have the UnicodeMath thing too which lets you type `\arrow` and the like, but for `->`, `<-`, `=>`, `<=`, `forall`, and `::` I set up auto-replace sequences.

It's the alignment thing that I really appreciate too!

**Comment by jdegoes** [comment]
We should have a team discussion to figure out "unicode policies" (since consistency is important to all of us, I think) and also share tips for inserting Unicode characters in the different editors we use.

**Comment by jonsterling** [comment]
@jdegoes Sounds great to me! After we've come to a consensus, would be nice to create a package like you suggested that contains the "standard" Unicode alternatives. And in the documentation for each symbol, it would be nice if there were included instructions on how to input that character in each major editor. (Or, alternatively, if suitable configuration files for major editors were available for download from that repository.)

**Comment by jdegoes** [comment]
> After we've come to a consensus, would be nice to create a package like you suggested that contains the "standard" Unicode alternatives. And in the documentation for each symbol, it would be nice if there were included instructions on how to input that character in each major editor.

:+1: :+1: :+1:

**Comment by jdegoes** [comment]
https://github.com/slamdata/purescript-gorgeous :smile:

**Comment by garyb** [comment]
Perhaps it'd be worth pulling `SlamData.Prelude` out as a separate module too, to share amongst our projects, and re-export `purescript-gorgeous` from there.

**Comment by jonsterling** [comment]
@garyb Perhaps so... I tend to be a little bit wary of preludes anyway, but it could be useful.

**Comment by garyb** [comment]
Yeah, I know what you mean, that's why I made it project specific initially.

It is rather tedious importing Maybe, Either, Tuple, `(=<<)`, `(*>)` in almost every module though :smile:

**Comment by jonsterling** [comment]
@garyb Yeah... Though it's less tedious when you qualify it like `import Maybe as M`, etc. as opposed to importing specifically what you will use. I don't really like to have almost anything pollute the global namespace, which is why I prefer to pay up front with imports. This way, it's always extremely easy to find the provenience of a function that is being used.

File: src/Quasar/QuasarF/Interpreter/Aff.purs [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+import Network.HTTP.StatusCode (StatusCode(..))
+
+import Quasar.Paths as Paths
+import Quasar.QuasarF (QuasarF(..), runLDJSON, AnyPath, Content, Pagination)
+
+type Config = { basePath ∷ AX.URL }
+
+eval
+  ∷ ∀ m eff
+  . ( MonadReader Config m
```
Comment (cryogenian) [inline src/Quasar/QuasarF/Interpreter/Aff.purs]: :+1:

Comment (garyb) [inline src/Quasar/QuasarF/Interpreter/Aff.purs]: I'll do something similar to handle authentication too, although `MonadState` I guess. I might need to put that in before I get anywhere with integrating this in SD actually.

**Pull request merged**